src/systems/combat.rs

```diff
@@ -83,7 +84,10 @@ impl<'s> System<'s> for PerformDefaultAttackSystem {
             }
 
             if let Some(value) = cooldown {
-                cooldowns.insert(event.attacker, value);
```

Comment:
Looking at the amethyst source code, it seems like an error cannot happen.
There it is implemented as `insert(....).expect("unreachable: We are inserting");` I think we should copy that pattern here.

Reply:
@sunreef would you like to make this change or should I merge as it is right now?

Reply:
@marot I changed it and commited.

Reply:
Thanks!